libkmod/libkmod-elf.c

```diff
@@ -54,9 +54,7 @@ struct kmod_elf {
 	} header;
 };
 
-//#define ENABLE_ELFDBG 1
-
-#if defined(ENABLE_LOGGING) && defined(ENABLE_ELFDBG)
+#if ENABLE_DEBUG
```

Comment:
Although I've first mentioned that I would prefer to have it enabled by default, I have understood now that it's a massive performance bottle neck. Having it in elf_get_uint breaks some of the optimizations which compilers could do, turning this into billions of calls on a default Arch Linux installation.

So, ENABLE_ELFDBG should stay special ...

Reply:
That still makes it special from my point of view. As long as it isn't activated for any of these distribution builds, I'm fine with it.

Reply:
> So, ENABLE_ELFDBG should stay special ...

correct - and the last time I enabled it for my own tests was maybe 10 years ago. We don't want all those strings to pollute the stderr. Should not not even the binary.... contrary to the debug messages, that is nice to enable from time to time when debugging e.g. libkmod with another tool.